js/templates/modals/settings/moderation.html

```diff
@@ -5,12 +5,15 @@ <h2 class="h3 clrT">Moderation</h2>
       <a class="btn clrP clrBr clrSh2 modalContentCornerBtn js-save"><%= ob.polyT('settings.btnSave') %></a>
     </div>
     <hr class="clrBr" />
+    <h3><%= ob.polyT('settings.moderationTab.disputeResolution') %></h3>
```

Comment:
Maybe an edge case, but since we have the space, should we make the width of the percentage field be able to support 2 digits and 2 decimal places without being cut-off?

![image](https://cloud.githubusercontent.com/assets/794517/22380039/b9b6174a-e470-11e6-802c-c867b3010a10.png)